src/javascript/track.js

```diff
@@ -77,19 +78,20 @@ export default class DropTrack {
 
   dropNote(key, keyObj) {
     if (key.includes(this.keyBind) && !this.vm.playMode) {
-      this.addNoteToArr(keyObj);
+      this.addNoteToArr(keyObj, null, this.game.checkHitLineY);
```

Comment:
Why is there an explicit `null` value here?

Reply:
To skip a parameter

Reply:
Any suggestions to make this more intuitive? make the parameter an object?

Reply:
You can use default function parameters I believe, since ECMAScript 2015.

```
function f(x = 1) {
    console.log(x);
}
```

Reply:
Ah yes, currently the default value is set inside the function, will rewrite this.